### PR TITLE
Add title tooltips for Projects and More menu items.

### LINKED ISSUE #866: Projects and "More" don't show a tooltip
The Projects and "More" dropdown in the main menu don't show a tooltip. 

screen shot 2016-12-06 at 10 18 49 am

### PR DESCRIPTION
**Pull request by microsoftsam**
Tooltips for Projects and More menu items. 

Projects: "Projects"
More (Cog icon): "More..."

@Jaqster, @pelikhan Happy with the titles for these?

 Fixes #866

### CODE REVIEW

**Comment by pelikhan** [comment]
A title should be more descriptive than the text otherwise leave it out.

File: webapp/src/app.tsx [hunk 1]
```diff
@@ -1810,11 +1810,11 @@ export class ProjectView extends data.Component<IAppProps, IAppState> {
                         {sandbox ? undefined : <div className="ui item landscape only"></div>}
                         {sandbox ? undefined : <div className="ui item widedesktop only"></div>}
                         {sandbox ? undefined : <div className="ui item widedesktop only"></div>}
-                        {sandbox ? undefined : <sui.Item class="openproject" role="menuitem" textClass="landscape only" icon="folder open" text={lf("Projects") } onClick={() => this.openProject() } />}
+                        {sandbox ? undefined : <sui.Item class="openproject" role="menuitem" textClass="landscape only" icon="folder open" text={lf("Projects") } title={lf("Projects")} onClick={() => this.openProject() } />}
```
Comment (pelikhan) [inline webapp/src/app.tsx]: title == text so not really useful.

Comment (microsoftsam) [inline webapp/src/app.tsx]: any thoughts on what this could be? or should we just leave it out?

**Comment by pelikhan** [comment]
Leave that one out for now.

Get Outlook for iOS<https://aka.ms/o0ukef>

________________________________

**Comment by microsoftsam** [comment]
Good to go.

**Pull request merged**